routes/web.php

```diff
@@ -44,7 +44,11 @@
     Route::post('settings/app_logo', SettingsController::class.'@storeAppLogo')->name('app.logo.change');
     Route::post('settings/app_name', SettingsController::class.'@storeAppName')->name('app.name.change');
 
-    Route::resource('bookings',\App\Http\Controllers\BookingRequestController::class)->only(['store', 'index', 'update', 'destroy']);
+    Route::resource('bookings',\App\Http\Controllers\BookingRequestController::class)->except('create');
+    Route::post('bookings/create',[\App\Http\Controllers\BookingRequestController::class, 'create'])->name('bookings.create');
+    Route::get('bookingsList',[\App\Http\Controllers\BookingRequestController::class, 'list'])->name('bookings.list');
```

Comment:
```suggestion
    Route::get('bookings/list',[\App\Http\Controllers\BookingRequestController::class, 'list'])->name('bookings.list');
```
Routes should be REST-ful